bot/exts/evergreen/status_cats.py

```diff
@@ -0,0 +1,33 @@
+import logging
+from http import HTTPStatus
+
+import discord
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+
+class StatusCats(commands.Cog):
+    """Commands that give statuses described and visualized by cats."""
```

Comment:
```suggestion
    """Commands that give HTTP statuses described and visualized by cats."""
```
Maybe be a bit more specific here saying they're HTTP statuses?